src/structure_managers/structure_manager_centers.cc

```diff
@@ -62,7 +72,7 @@ namespace rascal {
   // returns the number of cluster at Order=1, which is the number of atoms
   size_t StructureManagerCenters::get_nb_clusters(size_t order) const {
     if (order == 1) {
-      return this->natoms;
+      return this->n_center_atoms;
     } else {
       throw std::string("ERREUR : Order != 1");
```

Comment:
Just noticed the spelling... it reminds me of Molpro's `Global error FEHLER`, if you've ever had the joy of experiencing that message.